subsys/net/ip/connection.c

```diff
@@ -8,9 +8,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#define LOG_MODULE_NAME net_conn
 #define NET_LOG_LEVEL CONFIG_NET_CONN_LOG_LEVEL
 
+#include <logging/log.h>
+LOG_MODULE_REGISTER(net_conn, NET_LOG_LEVEL);
```

Comment:
what about using `CONFIG_NET_CONN_LOG_LEVEL` directly?

Reply:
I did it like this here are in other places you commented because LOG_LEVEL was not "found", I needed to check the current log level later in the code and did not want to use the the config symbol directly. Is it on purpose that LOG_LEVEL is not created by LOG_MODULE_REGISTER()?